hedera-mirror-rest/middleware/responseHandler.js

```diff
@@ -23,8 +23,10 @@ const {
   response: {headers},
 } = config;
 
-const contentTypeHeader = 'Content-Type';
-const applicationJson = 'application/json; charset=utf-8';
+const CONTENT_TYPE_HEADER = 'Content-Type';
+const APPLICATION_JSON = 'application/json; charset=utf-8';
+const LINK_NEXT_HEADER = 'Link';
+const linkNextHeaderValue = (linksNext) => `${linksNext}; rel="next"`
```

Comment:
Reading the spec again and it looks like we missed the angled brackets. Please double check the RFC.

```suggestion
const linkNextHeaderValue = (linksNext) => `<${linksNext}>; rel="next"`
```